src/applications/disability-benefits/all-claims/pages/conclusionAssault.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { PtsdNameTitle } from '../content/ptsdClassification';
+
+const conclusionDescription = (
+  <div>
+    <p>
+      Thank you for taking the time to answer our questions. The information you
+      provided will help us research your claim.
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': ({ formData }) => (
+    <PtsdNameTitle formData={formData} formType="781a" />
+  ),
+  'ui:description': conclusionDescription,
+};
+
+export const schema = {
```

Comment:
For pages with no schema, I'd recommend putting the schema directly into the form config, and the uiSchema as well if it's as simple as this one, with just a title and description. The content can go into `content/conclusionAssault.jsx`

For an example, take a look at the `newDisabilities` schema

Reply:
I can do that.

Reply:
So after talking to the team here, I think (and since you did otherwise approve this pr) to maintain code readability, I am going to leave these small configs as separate files.

We think the mixing and matching methodologies will be a net detriment to over all code readability.